xmr-stak.service

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=XMR-Stak Monero Miner
+After=network-online.target
+Wants=network-online.target
+AssertFileNotEmpty=/etc/xmr-stak/config.txt
+AssertFileNotEmpty=/etc/xmr-stak/pools.txt
+AssertFileNotEmpty=|/etc/xmr-stak/cpu.txt
+AssertFileNotEmpty=|/etc/xmr-stak/amd.txt
+AssertFileNotEmpty=|/etc/xmr-stak/nvidia.txt
+
+[Service]
+Type=simple
+Environment=GPU_FORCE_64BIT_PTR=1
+Environment=GPU_MAX_HEAP_SIZE=100
+Environment=GPU_USE_SYNC_OBJECTS=1
+Environment=GPU_MAX_ALLOC_PERCENT=100
+Environment=GPU_SINGLE_ALLOC_PERCENT=100
+WorkingDirectory=/etc/xmr-stak
+ExecStart=/usr/bin/xmr-stak
+DynamicUser=true
+Group=video
+CPUSchedulingPolicy=idle
+PrivateTmp=true
+ProtectHome=true
+ProtectSystem=strict
+NoNewPrivileges=true
+CapabilityBoundingSet=
+LimitMEMLOCK=256M
+MemoryDenyWriteExecute=true
```

Comment:
So this happens on the new ASM-enabled miner, unless you comment this line (allow memory write execute)
```
Sep 19 22:03:39 box8 xmr-stak[27087]: [2018-09-19 22:03:39] : Mining coin: cryptonight_v7
Sep 19 22:03:39 box8 xmr-stak[27087]: [2018-09-19 22:03:39] : Starting 1x thread, affinity: 0.
Sep 19 22:03:39 box8 xmr-stak[27087]: terminate called after throwing an instance of 'std::system_error'
Sep 19 22:03:39 box8 systemd[1]: xmr-stak.service: Main process exited, code=killed, status=6/ABRT
Sep 19 22:03:39 box8 systemd[1]: xmr-stak.service: Unit entered failed state.
Sep 19 22:03:39 box8 systemd[1]: xmr-stak.service: Failed with result 'signal'.
```